package.json

```diff
@@ -41,29 +43,31 @@
     "eslint-plugin-react": "7"
   },
   "peerDependencies": {
-    "autoprefixer": "7",
-    "babel-core": "6 || 7",
-    "babel-loader": "7",
-    "babel-plugin-transform-decorators-legacy": "1",
-    "babel-plugin-transform-react-constant-elements": "6",
-    "babel-preset-env": "1",
-    "babel-preset-react": "6",
-    "babel-preset-stage-0": "6",
-    "css-loader": "0.28",
-    "eslint-loader": "1",
-    "extract-text-webpack-plugin": "3",
-    "file-loader": "0.11",
-    "node-sass": "4",
-    "postcss-loader": "2",
-    "react": "^16.0.0-rc.3",
-    "react-dom": "^16.0.0-rc.3",
-    "sass-loader": "6",
-    "style-loader": "0.18",
-    "url-loader": "0.5",
-    "webpack": "3",
-    "webpack-dev-middleware": "1",
-    "webpack-dev-server": "2",
-    "webpack-isomorphic-tools": "3",
-    "whatwg-fetch": "2"
+    "autoprefixer": "^8.5.0",
+    "babel-core": "^6.26",
```

Comment:
consider switching to `@babel/*` packages for version 7 (although it's still in beta). @tomaskulich thoughts?